js/gemini.js

```diff
@@ -810,6 +831,39 @@ module.exports = class gemini extends Exchange {
         };
     }
 
+    async fetchDepositAddressesByNetwork (code, params = {}) {
+        await this.loadMarkets ();
+        const network = this.safeString (params, 'network');
+        if (!network) {
+            throw new BadRequest ('fetchDepositAddressByNetwork requires a network parameter');
```

Comment:
If we require something from the user in the arguments, then the exception type is `ArgumentsRequired`.
The exception message has this format:

```JavaScript
throw new Type (this.id + ' whichMethod() what');
```

So, this line should be

```JavaScript
throw new ArgumentsRequired (this.id + ' fetchDepositAddressByNetwork() requires a network parameter');
```

Reply:
That's more of the codestyling aspect.